.github/workflows/test.yml

```diff
@@ -19,9 +19,9 @@ jobs:
         uses: actions/setup-node@v3
         with:
           node-version: ${{ matrix.node-version }}
-      - uses: pnpm/action-setup@v2.2.4
+      - uses: pnpm/action-setup@v3.0.0
         with:
-          version: 7.19
+          version: 8.14
```

Comment:
It seems that pnpm 7.19 also works. Just curious, what is the reason for upgrading the pnpm version?

Reply:
IIRC, these two versions generate different `pnpm.lock` file, which will fail CI since CI checks if the git repo is clean after building

Reply:
Thanks for the explanation, I'll update the pnpm version